test/test_chemistry_rosetta.py

```diff
@@ -0,0 +1,48 @@
+from chemistry import read_PDB, load_rosetta
+from utils import get_fn
+import unittest
+from unittest import skipIf
+
+try:
+    from simtk.openmm.app import PDBFile
+except:
+    PDBFile = None
+    pass
+try:
+    from rosetta import init, pose_from_sequence
+except ImportError:
+    init = pose_from_sequence = None
+    pass
+
+
+def _unpackLen(obj):
+    return len(list(obj))
+
+
+class TestRosetta(unittest.TestCase):
+    """ Tests loading of a Rosetta pose object """
+
+    @skipIf(not init, "Cannot test load_rosetta module without PyRosetta.")
+    def testLoadPose(self):
```

Comment:
Add a docstring describing the test?  When you run `nosetests -vs test_chemistry_rosetta.py`, the docstring is printed followed by "ok" or "FAIL", and I like having that description, personally.